### PR TITLE
fix: suppress acceptance of done enquiries

### PR DESCRIPTION
**Pull request by webatspeed**

### CODE REVIEW

File: src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java [hunk 1]
```diff
@@ -37,6 +37,17 @@ public void verifySessionIsNotInProgress(ConsultantSessionDTO consultantSessionD
     }
   }
 
+  public void verifySessionIsNew(ConsultantSessionDTO consultantSessionDTO) {
+    if (conditionProvider.isNewSession(consultantSessionDTO.getSession())) {
```
Comment (maeventer) [inline src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java]: The conflict Eception should be thrown if the session is **not** new. As i read the condition in if statement it throws the conflict if the session is new. Therefore no new session (enquiry) could be assigned

File: src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java [hunk 2]
```diff
@@ -37,6 +37,17 @@ public void verifySessionIsNotInProgress(ConsultantSessionDTO consultantSessionD
     }
   }
 
+  public void verifySessionIsNew(ConsultantSessionDTO consultantSessionDTO) {
```
Comment (maeventer) [inline src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java:40]: Could you please also cover this method directly with additional test

**Review by maeventer — CHANGES_REQUESTED** [review]

File: src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java (hunk 2 above)
Comment (webatspeed) [reply src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java:40]: two tests added

File: src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java (hunk 1 above)
Comment (webatspeed) [reply src/main/java/de/caritas/cob/userservice/api/facade/assignsession/SessionToConsultantVerifier.java]: naturally...

**Review by maeventer — APPROVED** [review]

**Review by hill-daniel — APPROVED** [review]

**Pull request merged**